app/models/answer.rb

```diff
@@ -26,17 +26,17 @@ def likers
 
   def answer_notify(current_user)
     # notify question author
-    if current_user.uid != self.author.uid
-      AnswerMailer.notify_question_author(self.author, self).deliver
+    if current_user.uid != self.node.author.uid
```

Comment:
Earlier, the if conditional `current_user.uid != self.author.uid` was always false since the current_user was always the the answer's author. I have changed that to the node's author.

> I think we do want to notify original node (question) author when an answer is posted

Agreed. Line 30 sends the mail to the node's author.

Thank you very much for reviewing.